MapboxDirections/MBRouteOptions.swift

```diff
@@ -304,6 +312,13 @@ open class RouteOptions: NSObject, NSSecureCoding {
     open var includesExitRoundaboutManeuver = false
     
     /**
+     An Array of `RoadClass` that the routing engine will attempt to avoid.
```

Comment:
RoadClasses is an OptionSet, not an Array. OptionSets just happen to use the same `[]` syntax, but it’s perfectly valid to set `intersection.outletRoadClasses = .toll`, whereas that would be invalid if `roadClasses` were of type `[RoadClasses]`.

Reply:
Presently we do not support more than one road class. You may avoid one or another, but not more than one.

Reply:
If there’s the potential that the API might support more than one excluded road class in the future, let’s leave the code as is but change the documentation to avoid using the term “array” and assert at runtime that only one is set. Otherwise, if there’s a fundamental reason why the API wouldn’t support more than one, then I guess we could consider a separate enumeration that doesn’t bridge as an OptionSet, but that would be much less elegant.

Reply:
How about:

> The road classes that the resulting routes will avoid.
> 
> For example, you can set this property to `.toll` to avoid toll roads. No road classes are avoided by default. Currently, it is only possible to avoid one road class at a time.